docs/getting-started/devenv/using-visual-studio.md

```diff
@@ -1,13 +1,229 @@
-# 🔧 Development using Visual Studio 2015
-
-> **Note**
-> 
-> This topic hasn’t been written yet! 
->
-> We welcome your input to help shape the scope and approach. You can track the status and provide input on this
-> [issue](https://github.com/dotnet/core-docs/issues/281) at GitHub.
-> 
-> If you would like to review early drafts and outlines of this topic, please leave a note with your contact information in the issue.
->
-> Learn more about how you can contribute on [GitHub](https://github.com/dotnet/core-docs/blob/master/CONTRIBUTING.md).
->
+# Development using Visual Studio 2015
+
+by [Bertrand Le Roy](https://github.com/bleroy) and [Phillip Carter](https://github.com/cartermp)
+
+Visual Studio 2015 provides a full-featured development environment for developing .NET Core applications. The scripts in this document describe the steps necessary to build a number of typical .NET Core solutions, or solutions that include .NET Core components, using Visual Studio. The scenarios include testing and using third-party libraries that have not been explicitly built for the most recent version of .NET Core.
```

Comment:
@bleroy @cartermp Is there a reason you used "scripts" here? Can we change to "procedures'?